examples/blender/blender.py

```diff
@@ -45,7 +45,7 @@ async def worker(ctx: WorkContext, tasks):
         package=package,
         max_workers=10,
         budget=10.0,
-        timeout=timedelta(minutes=15),
+        timeout=timedelta(minutes=5),
```

Comment:
This value either needs to be extended or tweaked in https://github.com/golemfactory/yagna/pull/480/files#diff-1f4b28d81b374dbab50450158491edccR83-R85

Reply:
I've updated the timeout to be derived from the number of frames to render (see: faf95512ac691d32d0c9ae90df65c13edab1ffcb).